src/functions/advanced-functions/event.php

```diff
@@ -103,19 +103,25 @@ function tribe_create_event( $args ) {
 	}
 
 	/**
-	 * Update an Event.
+	 * Update an Event using the legacy method.
+	 *
+	 * Note: This function is outdated and should be replaced with the [TEC ORM `tribe_events()->save()` method](https://docs.theeventscalendar.com/apis/orm/basics/#save).
 	 *
 	 * @category Events
 	 *
 	 * @link     http://codex.wordpress.org/Function_Reference/wp_update_post
 	 *
 	 * @see      wp_update_post()
 	 * @see      tribe_create_event()
+	 * @see      Tribe__Repository::save()
 	 *
 	 * @param int|bool   $postId  ID of the event to be modified.
 	 * @param array      $args    Args for updating the post. See {@link tribe_create_event()} for more info.
 	 *
 	 * @return int|bool ID of the event that was created. False if update failed.
+	 *
+	 * @version TBD
+	 * @since   TBD Updated docblock to indicate future deprecation of function.
```

Comment:
Since 3.0.0 no need to mark Docblock changes on the since
```suggestion
	 * @since   3.0.0
```